RiotSharp/RiotApi.cs

```diff
@@ -212,37 +212,30 @@ public Champion GetChampion(Region region, int championId)
         public Dictionary<long, List<MasteryPage>> GetMasteryPages(Region region, List<long> summonerIds)
```

Comment:
Since the api only supports one summoner id per request, I wouldn't differ from Riot's implementation. As partially discussed in #371, RiotSharp is a stupid wrapper and we shouldn't extend the features of the api interface (even if this is only virtually).